default/python/AI/FreeOrionAI.py

```diff
@@ -64,7 +64,7 @@ def startNewGame(aggression_input=fo.aggression.aggressive):  # pylint: disable=
     Should clear any pre-existing state and set up whatever is needed for AI to generate orders."""
     empire = fo.getEmpire()
     if empire is None:
-        print "This client has no empire. Ignoring new game start message."
+        info("This client has no empire. Ignoring new game start message.")
```

Comment:
This seems like enough of an unexpected situation that it should be debug or warn rather than merely info.

Reply:
> it should be debug or warn

If we can't get an empire after just creating a game, it seems we should log as ```fatal``` and terminate the AI process to be honest.

Reply:
I will set it fatal but will not terminate anything,  this AI log will have only this messages, so it does not matter what error level we use.